server/routerlicious/pnpm-lock.yaml

```diff
@@ -12299,6 +12299,21 @@ packages:
       '@mongodb-js/saslprep': 1.1.0
     transitivePeerDependencies:
       - aws-crt
+    dev: true
+
+  /mongodb/4.17.1:
```

Comment:
It looks like 4.17.0 wasn't removed. Only 4.17.1 was *added*. So that means we still have a dependency on 4.17.0 somewhere.

Reply:
Ah I see, it's just above here which is why `dev: true` was added. It's a dev dependency through type tests.